docs/reference/es_compatible_api.md

```diff
@@ -366,7 +366,32 @@ The following query types are supported.
 | `query`           | String     |  Full-text search query. The last token will be prefix-matched   | -       |
 | `zero_terms_query`|  `all` or `none` | Defines if all (`all`) or no documents (`none`) should be returned if the query does not contain any terms after tokenization. | `none` |
 | `max_expansions`  |  `Integer` | Number of terms to be match by the prefix matching.               |  50     |
-| `slop`            | `Integer`  | Allows extra tokens between the query tokens.                    |  0      |
+| `analyzer`        | String     | Analyzer meant to cut the query into terms. It is recommended to NOT use this parameter. |  The actual field tokenizer.  |
```

Comment:
I think we don't use the analyzer set in the query in the code. 

See the code in the main branch: 

https://github.com/quickwit-oss/quickwit/blob/main/quickwit/quickwit-query/src/query_ast/full_text_query.rs#L98-L99

And I guess we don't test this.

I'm ok to merge this PR and open issue to fix that or the docs.

Reply:
I think we do use in the body of that method. Let me add a test for this.

Reply:
I added a shitty test for it. It at least checks that it is not no-op.